### PR TITLE
Install OpenCV and Pandas in docker container for jupyter

### PR DESCRIPTION
**Pull request by lossyrob**
This PR makes the jupyter container its own container, that installs opencv and pandas.

Testing instructions:
- ssh into provisioned vagrant box
- `./scripts/update --jupyter`
- `./scripts/jupyter`

You will now have a notebook that has access to the pandas and cv2 packages.

### CODE REVIEW

**Comment by lewfish** [comment]
Is this ready for review?

**Comment by lossyrob** [comment]
Yes ready for review. Is there a procedure on this repo for reviews?

**Comment by lewfish** [comment]
On Civic Apps, we assign a reviewer when the PR is ready for review. We also include testing instructions. I'm okay with only including the instructions if it's non-obvious. I don't think we need any procedure beyond that at this point.

**Comment by lewfish** [comment]
When it gets approved, the PR author then merges and deletes the branch. At this point, I think people should add me as the reviewer. As for my PRs, I think we can just keep them unreviewed to keep things moving along quickly. Unless you want to review them first.

**Comment by lossyrob** [comment]
Ok, I'm fine with that process. I agree about your PRs for keeping things moving, but feel free to assign me or Annie if you want a second pair of eyes.

**Comment by lewfish** [comment]
I'll make sure to assign Annie to some simple PRs starting in a week or two so she gets the experience.

File: Vagrantfile [hunk 1]
```diff
@@ -11,10 +11,15 @@ SCRIPT
 
 DATA_DIR = ENV["RASTER_VISION_DATA_DIR"]
 if DATA_DIR.nil?
-  puts "You must set the environment variable: RASTER_VISION_DATA_DIR"
+  puts "ERROR: You must set the environment variable: RASTER_VISION_DATA_DIR"
```
Comment (lewfish) [inline Vagrantfile]: The changes in this file are already on the develop branch along with a bug fix at https://github.com/azavea/raster-vision/blob/develop/Vagrantfile#L34

Comment (lewfish) [inline Vagrantfile]: Oh nm, I see the next commit.

**Comment by lossyrob** [comment]
@lewfish updated README and merge in develop.

**Comment by lewfish** [comment]
I don't see the updated README.

**Comment by lossyrob** [comment]
Ah. Confused myself with the process - I made this PR off of my fork, not the branch on this repo. Should be there now.

**Comment by lossyrob** [comment]
Ok, I'll do rebasing moving forward if we aren't doing merge commits in this repository; rather then wrestle with these I'll merge them in this time.

**Pull request merged**